rust/sbp/src/lib.rs

```diff
@@ -154,14 +154,16 @@ mod tests {
 
     #[test]
     fn making_frame() {
+        use crate::messages::SBPMessage;
+
         let msg = crate::messages::system::MsgStartup {
             sender_id: Some(250),
             cause: 1,
             startup_type: 45,
             reserved: 0,
         };
 
-        let frame = crate::framer::to_frame(&msg).unwrap();
+        let frame = msg.to_frame().unwrap();
```

Comment:
the whole point I guess is just this little bit of syntactic sugar 🤷‍♂

Reply:
Yup! I'm all for sugar that simplifies code!